src/reactions/block.json

```diff
@@ -18,12 +18,7 @@
 			}
 		}
 	},
-	"attributes": {
```

Comment:
This seems problematic as it will create validation errors for existing users of the block, whenever they try to visualize the block in the site editor.
I would recommend implementing [deprecation](https://developer.wordpress.org/block-editor/reference-guides/block-api/block-deprecation/) to avoid that.